src/shared/constants/chart-props.js

```diff
@@ -38,6 +41,7 @@ export const chartDefaultProps = {
   axisLeftLabelDisplayFn: d => d,
   width: 100,
   height: 100,
+  legendProps: { 'maxRowLegendItems': MAX_LEGEND_ITEMS_ROW }
```

Comment:
_pref:_  `legendProps` is passed directly to the Nivo legend through `{...legendProps}`, which overrides all other keys. It was first used as an internal way to customize the legend for different charts, eventually only supporting BarChart `dataFrom`.

I think exposing it here is unnecessary and opens the door for a user to mistakenly break something (e.g. they can pass `direction: 'column'` or `anchor` props). We've built a lot of "opinionated" functionality surrounding the legend, so I think this would be more clear as a flat prop. Alternatively, you could move the `{...legendProps}` higher so it doesn't override our internal logic.

Reply:
Fixed